test/machines/machines_test.go

```diff
@@ -326,7 +327,7 @@ var _ = framework.SigKubeDescribe("Machines", func() {
 			if workerMachineSet.Annotations == nil {
 				workerMachineSet.Annotations = map[string]string{}
 			}
-			workerMachineSet.Annotations[machineutils.ExcludeNodeDrainingAnnotation] = ""
+			workerMachineSet.Annotations[machinecontroller.ExcludeNodeDrainingAnnotation] = ""
```

Comment:
Commenting on the line below because that's all github will let me do - can we drop the following printf/

Reply:
I am going to kill this file completely eventually and replace it with the testsuite we have. Will do it then :).